packages/ipxe-bluebanquise/bluebanquise_dhcpretry.ipxe

```diff
@@ -1,30 +1,9 @@
-#!ipxe
-
-echo . . . . . . . . . . . . . :::!~!!!!!:. . . . . . . . .
-echo . . . . . . . . . . . .xUHWH!!.!!?M88WHX:. . . . . . .
-echo . . . . . . . . . . .X*#M@$!!. !X!M$$$$$$WWx:. . . . .
-echo . . . . . . . . . .:!!!!!!?H!.:!$!$$$$$$$$$$8X:. . . .
-echo . . . . . . . . . !!~. ~:~!!.:~!$!#$$$$$$$$$$8X:. . ..
-echo . . . . . . . . .:!~::!H!<. .~.U$X!?R$$$$$$$$MM!. . ..
-echo . . . . . . . . .~!~!!!!~~. :XW$$$U!!?$$$$$$RMM!. . ..
-echo . . . . . . . . . .!:~~~. :!M"T#$$$$WX??#MRRMMM!. . ..
-echo . . . . . . . . . .~?WuxiW*`. .`"#$$$$8!!!!??!!!. . ..
-echo . . . . . . . . .:X-.M$$$$. . . .`"T#$T~!8$WUXU~. . ..
-echo . . . . . . . . :%`. ~#$$$m:. . . . ~!~.?$$$$$$. . . .
-echo . . . . . . . :!`.-. .~T$$$$8xx. . xWW-.~""##*". . . .
-echo . . . . . . -~~:<`.!. . ~?T#$$@@W@*?$$. . . /`. . . ..
-echo . . W$@@M!!!. !~~.!!. . . :XUW$W!~.`"~:. . :. . . . ..
-echo . . M"~~`.:x%`!!. !H:. .!WM$$$$Ti.:. !WUn+!`. . . . ..
-echo . . :::~:!!`:X~. :.?H.!u."$$$B$$$!W:U!T$$M~. . . . . .
-echo . . .~~. .:X@!.-~. .?@WTWo("*$$$W$TH$!.`. . . . . . ..
-echo . . Wi.~!X$?!-~. . :.?$$$B$Wu("**$RM!. . . . . . . . .
-echo . . $R@i.~~.!. . .:. .~$$$$$B$$en:``. . . . . . . . ..
-echo . . ?MXT@Wx.~. . :. . .~"##*$$$$M~. . . . . . . . . ..
+
 echo
-echo +----------------------------------------------------+
-echo | . . . . . . . iPXE embed - dhcpretry . . . . . . . |
-echo | . . . . . BlueBanquise - Benoit Leveugle . . . . . |
-echo +----------------------------------------------------+
+echo . . . +----------------------------------------------------+ . . .
+echo . . . | . . . . . . . iPXE embed - standard. . . . . . . . | . . .
```

Comment:
Change from dhcpretry to standard in file bluebanquise_dhcpretry.ipxe: is it expected?

Reply:
Good catch. Solved.